### PR TITLE
[SFEQS-686] Scaffold io-functions-sign repository

### PR DESCRIPTION
**Pull request by lucacavallaro**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "env": {
+    "es2021": true,
+    "node": true
+  },
+  "extends": ["@pagopa/eslint-config/recommended"],
+  "rules": {
+    "functional/immutable-data": "off"
```
Comment (gunzip) [inline .eslintrc.json]: why off?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+node_modules
```
Comment (gunzip) [inline .gitignore:1]: I'd add at least 

```
.env*

*.js
!jest.config.js
!.eslintrc.js

```

File: .node-version [hunk 3]
```diff
@@ -0,0 +1 @@
+16.13.0
```
Comment (gunzip) [inline .node-version:1]: does this match the deployed function runtime version?

File: docs/openapi.yaml [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+        schema:
+          type: string
+          format: uuid
+    get:
+      operationId: GetSigner
+      tags:
+        - Signer
+      responses:
+        200:
+          description: Il profilo del Signer
```
Comment (gunzip) [inline docs/openapi.yaml]: should be in english see https://github.com/pagopa/io-functions-services/blob/master/openapi/index.yaml

File: host.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+  "version": "2.0",
+  "extensionBundle": {
```
Comment (gunzip) [inline host.json]: we usually prefer to control packages version though csjprog

https://github.com/pagopa/io-functions-services/blob/master/extensions.csproj

**Review by gunzip — CHANGES_REQUESTED** [review]

File: .node-version (hunk 3 above)
Comment (lucacavallaro) [reply .node-version:1]: yes

**Review by gunzip — APPROVED** [review]

**Pull request merged**